app/src/main/res/layout/selection_interaction_item.xml

```diff
@@ -41,6 +41,9 @@
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="14sp"
       android:textStyle="italic"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
```

Comment:
The text should not be `centre` aligned. Instead it should be `start` aligned.

Reply:
Done.